packages/notifier/test/test-publish-kit.js

```diff
@@ -1,274 +1,476 @@
 /* eslint-disable @typescript-eslint/prefer-ts-expect-error -- https://github.com/Agoric/agoric-sdk/issues/4620 */
 
-import { test } from './prepare-test-env-ava.js';
+import { test } from '@agoric/swingset-vat/tools/prepare-test-env-ava.js';
+import { E } from '@endo/eventual-send';
+import { makeMarshal } from '@endo/marshal';
+import {
+  buildKernelBundles,
+  initializeSwingset,
+  makeSwingsetController,
+} from '@agoric/swingset-vat';
+import { provideHostStorage } from '../../SwingSet/src/controller/hostStorage.js';
+// import { makeScalarMapStore as makeBaggage } from '@agoric/store';
+import { makeScalarBigMapStore } from '../../vat-data/src/vat-data-bindings.js';
```

Comment:
this references another package so it should use package loading, not file system paths which may change.
```suggestion
import { provideHostStorage } from '@agoric/swingset-vat/src/controller/hostStorage.js';
import { makeScalarBigMapStore } from '@agoric/vat-data/src/vat-data-bindings.js';
```

Reply:
Done, except for `bfile('../../SwingSet/test/bootstrap-relay.js')` (which I don't know how to fix because `bfile` relies on `new URL`).